src/CoreText/CTFontCollection.cs

```diff
@@ -178,6 +178,22 @@ public CTFontDescriptor[] GetMatchingFontDescriptors ()
 			return matches;
 		}
 
+		[Mac (10,7), iOS (12,0), TV (12,0), Watch (5,0)]
+		[DllImport (Constants.CoreTextLibrary)]
+		static extern IntPtr CTFontCollectionCreateMatchingFontDescriptorsWithOptions (IntPtr collection, IntPtr options);
+
+		[Mac (10,7), iOS (12,0), TV (12,0), Watch (5,0)]
+		public CTFontDescriptor [] GetMatchingFontDescriptors (CTFontCollectionOptions options)
+		{
+			var cfArrayRef = CTFontCollectionCreateMatchingFontDescriptorsWithOptions (handle, options == null ? IntPtr.Zero : options.Dictionary.Handle);
```

Comment:
minor: we have an extension method `GetHandle ()` that does the null check (to return `IntPtr.Zero`)

Reply:
That doesn't work here because the code is not doing `options.Handle`, but `options.Dictionary.Handle`.